components/lib/utils/DomHandler.js

```diff
@@ -1055,9 +1055,9 @@ export default class DomHandler {
 
     static applyStyle(element, style) {
         if (typeof style === 'string') {
-            element.style.cssText = this.style;
+            element.style.cssText = style;
```

Comment:
@melloware I updated `DomHandler.applyStyle` to correctly "apply" the passed in `style` parameter.